src/forcedloaddialog.cpp

```diff
@@ -25,15 +25,51 @@ void ForcedLoadDialog::setValues(QList<MOBase::ExecutableForcedLoadSetting> &val
 {
   ui->tableWidget->clearContents();
 
-  for (int i = 0; i < values.count(); i++) {
+  for (auto forcedLoad : m_GamePlugin->executableForcedLoads())
+  {
     ForcedLoadDialogWidget *item = new ForcedLoadDialogWidget(m_GamePlugin, this);
-    item->setEnabled(values[i].enabled());
-    item->setProcess(values[i].process());
-    item->setLibraryPath(values[i].library());
-    item->setForced(values[i].forced());
+    item->setEnabled(false);
+    item->setProcess(forcedLoad.process());
+    item->setLibraryPath(forcedLoad.library());
+    item->setForced(true);
+
+    int row = ui->tableWidget->rowCount();
+    ui->tableWidget->insertRow(row);
+    ui->tableWidget->setCellWidget(row, 0, item);
+  }
 
-    ui->tableWidget->insertRow(i);
-    ui->tableWidget->setCellWidget(i, 0, item);
+  for (int i = 0; i < values.count(); i++) {
+    // Search for a similar forced load
+    int row = 0;
+    for (; row < ui->tableWidget->rowCount(); row++)
+    {
```

Comment:
I know it's not clear but I think most of MO2 uses opening brackets on the same line.

Reply:
No longer relevant